index.js

```diff
@@ -1913,24 +1913,13 @@ Auth0.prototype.getSSOData = function (withActiveDirectories, cb) {
 /**
  * Get all configured connections for a client
  *
- * @example
- *
- *     auth0.getConnections(function (err, conns) {
- *       if (err) return console.log(err.message);
- *       expect(conns.length).to.be.above(0);
- *       expect(conns[0].name).to.eql('Apprenda.com');
- *       expect(conns[0].strategy).to.eql('adfs');
- *       expect(conns[0].status).to.eql(false);
- *       expect(conns[0].domain).to.eql('Apprenda.com');
- *       expect(conns[0].domain_aliases).to.eql(['Apprenda.com', 'foo.com', 'bar.com']);
- *     });
- *
  * @method getConnections
  * @param {Function} callback
+ * @deprecated This method is deprecated. If you need to get the connections please use Management API https://auth0.com/docs/api/management/v2#!/Connections/get_connections
```

Comment:
@hzalaz Is there an new auth0.js method that calls the v2 management API (an equivalent to this now deprecated method)?